docs/components/search-bar.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebarDepth: 0
+---
+
+# SearchBar
+
+This is an overview of the most common usage of `<SearcBar>`.
+
+`<SearchBar>` is a UI component that provides a user interface for entering search queries and submitting requests to the search provider.
+
+<DocExampleBox codeBox="https://codesandbox.io/s/m7yz90mn6p?module=%2Fsrc%2FApp.vue">
+```html
+<SearchBar hint="Search hint" :text="searchPhrase" />
+```
+```js
+onSubmit(){
+    alert("Searching...");
+}
+```
+<SearchBarDoc />
+</DocExampleBox>
+
+`<SearchBar>` provides two-way data binding using v-model.
+
+```html
+<SearchBar v-model="searchQuery" />
+```
+
+## Props
+
+| Name                     | Type   | Default | Web | Mobile |
+| ------------------------ | ------ | ------- | --- | ------ |
+| hint                     | String |         | ✔   | ✔      |
```

Comment:
It would be nice to add a description column for both props and events

Reply:
done